activation/atx_pool.go

```diff
@@ -13,13 +13,24 @@ type AtxMemPool struct {
 	atxMap map[types.ATXID]*types.ActivationTx
 }
 
+// GetEpochAtxs returns mock atx list
+func (mem *AtxMemPool) GetEpochAtxs(epochID types.EpochID) (atxs []types.ATXID) {
```

Comment:
I don't understand why this is needed here, this is not a mock.

Reply:
AtxMemPool is now redundant and only used for tests

Reply:
if we test using it as a memory db so ok (which can be replaced with a regular in-mem db). otherwise I don't get why we need it for testing.